src/sp/it/pl/util/file/Util.kt

```diff
@@ -16,10 +14,9 @@ import kotlin.streams.asStream
 private val logger = KotlinLogging.logger { }
 
 /**
- * Returns [File.getName] or [File.toString] if this is the root directory,
- * since that returns an empty string otherwise.
+ * [File.getName], which returns partition name for root directories.
```

Comment:
```suggestion
 * @return [File.getName], but partition name for root directories
```

Otherwise it looks like an alias `File.getName`